gradle.properties

```diff
@@ -5,10 +5,13 @@ bintrayRepoType=defineMe
 
 eventuateMavenRepoUrl=https://dl.bintray.com/eventuateio-oss/eventuate-maven-release/,https://dl.bintray.com/eventuateio-oss/eventuate-maven-rc/,file:///Users/cer/.m2/testdeploy
 
-springBootVersion=2.0.3.RELEASE
-eventuateClientVersion=0.20.1.RELEASE
-eventuateTramVersion=0.20.0.RELEASE
-eventuateLocalVersion=0.30.0.RELEASE
+springBootVersion=2.1.1.RELEASE
+eventuateClientVersion=0.20.0-SNAPSHOT
+eventuateTramVersion=0.20.0.REDIS-SNAPSHOT
+eventuateLocalVersion=0.15.0-SNAPSHOT
+#eventuateTramImageVersion=$eventuateTramVersion
+eventuateTramImageVersion=0.21.1.RELEASE
```

Comment:
Aren't there other image versions now?

Reply:
Yes, it should be eventuate-cdc image. 
And I will use released version.